test/parallel/alice.js

```diff
@@ -0,0 +1,13 @@
+// Alice sends a message to Bob for Bob's eyes only.
+'use strict';
```

Comment:
tests should be named as `test-{module}-{short description}` such as `test-zlib-deflate-raw-inherits.js` and files that are dependencies of tests should be equally well named and moved into the relevant fixtures folder.

Reply:
parallel/test-frenemies.js -> parallel/test-frenemies
parallel/{alice,bob,carol}.js -> fixtures/module-frenemies/box-and-unbox/